iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/GeneratorScalable.java

```diff
@@ -126,7 +126,7 @@ public double scale(Network n, double asked, ScalingParameters parameters) {
         double oldTargetP = g.getTargetP();
         double minimumTargetP = minimumTargetP(g);
         double maximumTargetP = maximumTargetP(g);
-        if (oldTargetP < minimumTargetP || oldTargetP > maximumTargetP) {
+        if (!parameters.isAllowOutOfBoundsGeneratorTargetP() && (oldTargetP < minimumTargetP || oldTargetP > maximumTargetP)) {
```

Comment:
After a talk with @geofjamg we prefer `parameters.allowsGeneratorOutOfActivePowerLimits()`.

Reply:
Renamed